boards/riscv/tlsr9518adk80d/Kconfig.defconfig

```diff
@@ -55,4 +55,11 @@ DT_CHOSEN_Z_CODE_PARTITION := zephyr,code-partition
 config FLASH_LOAD_OFFSET
 	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_CODE_PARTITION)) if USE_DT_CODE_PARTITION
 
+# Buffer for image writter shall be less(multiple of access alignment) or
+# equal to flash page. tlsr9518adk80d boards use external P25Q16 IC as
+# flesh memory. Flash page size of the IC is 256 bytes. So that, it is
+# maximum image writer buffer size for such kind of boards.
+# config IMG_BLOCK_BUF_SIZE
+#	default 256 if MCUBOOT_IMG_MANAGER
```

Comment:
Why is this commented out?

Reply:
Hi @fkokosinski 

It were requested to comment it out by reviewers in https://github.com/zephyrproject-rtos/zephyr/pull/45973/files

I may remove it if the comment is confusing.

Regards,
Alex

Reply:
Where exactly? I don't see the comment that you're referring to.

I just don't get why would you put a conditional defconfig and then later comment it out. I think that you should either correct the condition (or the symbol value when the condition is met), or remove it altogether.

Reply:
Oh, hell... Sorry, I've forgot to uncomment it :) My mistake.